test/runtests.jl

```diff
@@ -1,10 +1,24 @@
+# path=pwd()
+# if path[end-6:end] != "MatLang"
+# 	error("cd(to the MatLang path)")
+# end
+
+push!(LOAD_PATH, ".")
+using Revise
 using MatLang
 using Test
 
-# usage
-clcM()
-
 # tests
 @testset "MatLang.jl" begin
-1==1
+
+    @test eyeM(2,3)==[1 0 0; 0 1 0]
+    @test diagM([2,3,4],1)==[0 2 0 0; 0 0 3 0; 0 0 0 4; 0 0 0 0]
+end
+
+
+@testset "# Native Julia functions that are just a copy" begin
```

Comment:
"# Native Julia functions that are just a copy" is quite a long title to me.

Reply:
Agree. If you have a better replacement I would appreciate it. 😄